app/controllers/reset_passwords_controller.rb

```diff
@@ -0,0 +1,44 @@
+class ResetPasswordsController < ApplicationController
+  skip_before_action :authenticate_user
+  before_action :ensure_no_user
+  before_action :pundit_authorize_user
```

Comment:
that would mean I should completely remove pundit from reset password functionality (so get rid of `reset_password_policy.rb` too) ?